connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/DistributedHerder.java

```diff
@@ -113,6 +113,8 @@
 
     private final ExecutorService forwardRequestExecutor;
 
+    private Map<String, Connector> tempConnectors = new HashMap<>();
```

Comment:
Same question as before -- doesn't seem like this should be here since we have the same in `AbstractHerder` and I'm not even clear what the one use of `tempConnectors` in this file accomplishes.